crates/sui-indexer/src/metrics.rs

```diff
@@ -63,36 +65,50 @@ impl IndexerCheckpointHandlerMetrics {
                 registry,
             )
             .unwrap(),
-            fullnode_download_latency: register_histogram_with_registry!(
-                "checkpoint_full_node_read_request_latency",
+            fullnode_checkpoint_download_latency: register_histogram_with_registry!(
+                "fullnode_checkpoint_download_latency",
                 "Time spent in waiting for a new checkpoint from the Full Node",
                 LATENCY_SEC_BUCKETS.to_vec(),
                 registry,
             )
             .unwrap(),
+            fullnode_transaction_download_latency: register_histogram_with_registry!(
+                "fullnode_transaction_download_latency",
+                "Time spent in waiting for a new transaction from the Full Node",
+                LATENCY_SEC_BUCKETS.to_vec(),
+                registry,
+            )
+            .unwrap(),
+            fullnode_object_download_latency: register_histogram_with_registry!(
+                "fullnode_object_download_latency",
+                "Time spent in waiting for a new epoch from the Full Node",
+                LATENCY_SEC_BUCKETS.to_vec(),
+                registry,
+            )
+            .unwrap(),
             checkpoint_index_latency: register_histogram_with_registry!(
-                "checkpoint_index_request_latency",
+                "checkpoint_index_latency",
```

Comment:
this will break the dashboard again right? Should we just keep the original naming?

Reply:
I will update the dash myself after next deployment, it's easy

Reply:
to make the dash show the new metrics, we will need to update the dash anyways